test/suite/test_compact11.py

```diff
@@ -163,24 +164,32 @@ def test_compact11(self):
         # Turn on background compaction to allow the bitmap blocks to be modified
         # from compact operation.
         self.turn_on_bg_compact('free_space_target=1MB')
+
+        bytes_recovered = 0
         while self.get_files_compacted(uris) < self.num_tables:
-            time.sleep(0.5)
+            new_bytes_recovered = self.get_bytes_recovered()
+            if new_bytes_recovered != bytes_recovered:
+                # Update the incremental backup ID from the parent class.
+                self.bkup_id += 1
+                shutil.copytree(self.home_tmp, self.backup_incr + str(self.bkup_id))
+                self.take_incr_backup(self.backup_incr + str(self.bkup_id), 0, self.bkup_id)
```

Comment:
Is it on purpose to always use `0` as the source? Is it because we change directory?

Reply:
I think it is because of my comment above to be able to compare during the loop and have a known base